projects/search-admin/docker-compose.yml

```diff
@@ -46,6 +47,7 @@ services:
       BINDING: 0.0.0.0
       PORT: 3000
       DISCOVERY_ENGINE_DEFAULT_COLLECTION_NAME: "projects/780375417592/locations/global/collections/default_collection"
+      PUBLISHING_API_BEARER_TOKEN: example
```

Comment:
Similar question here. Is this needed when the app already "depends on" `publishing-api-app`?

I'm comparing it to the set-up for the other publishing apps. e.g. https://github.com/alphagov/govuk-docker/blob/main/projects/collections-publisher/docker-compose.yml#L40